src/branchandbound.jl

```diff
@@ -19,7 +19,8 @@ function _branch_bound(bab::BranchAndBoundEnclosure, f::Function, X::Interval_or
 
     fX = f(X)  # TODO: allow user to choose how to evaluate this (mean value, natural enclosure)
     # if tolerance or maximum number of iteration is met, return current enclosure
-    if diam(fX) <= bab.tol || cnt == bab.maxdepth
+    min_abs = in_interval(0, fX) ? zero(fX.lo) : min(abs(fX.lo), abs(fX.hi))
```

Comment:
This is the definition of mignitude

```suggestion
    min_abs = mig(fX)
```

Reply:
Nice. Then I suggest to inline the function instead of having this extra line.